manim/mobject/mobject.py

```diff
@@ -125,22 +125,33 @@ def __repr__(self):
         return str(self.name)
 
     def reset_points(self):
+        """
```

Comment:
I will suggest you to read https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP) first.

Reply:
Ok, I fixed the line break and type specification issues :)